scripts/unlock-package-lock.js

```diff
@@ -2,25 +2,34 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-const fs = require('fs');
-const path = require('path');
+const fs = require('node:fs');
+
+const lockfiles = [
+  'package-lock.json',
+  'backstop/package-lock.json',
```

Comment:
This file does not have dependencies on our components, but I think having an index of all lockfiles in the repository is useful.